var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -33,7 +34,7 @@ class Namd(MakefilePackage, CudaPackage):
     version(
         "2.14",
         sha256="34044d85d9b4ae61650ccdba5cda4794088c3a9075932392dd0752ef8c049235",
-        preferred=True,
+        default=True,
```

Comment:
It used to say "preferred=True" and now it says "default=True", which the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-selection don't mention as a keyword that I can find.

Reply:
Yep, sorry. You are blazing fast at reviewing and realised my silly mistake before me ;)